### PR TITLE
Use `wp (plugin|theme) update --format=summary` for one-liner messages

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #1934

### CODE REVIEW

File: php/WP_CLI/CommandWithUpgrade.php [hunk 1]
```diff
@@ -264,18 +264,24 @@ protected function update_many( $args, $assoc_args ) {
...
-			$status = array();
-			foreach($items_to_update as $item_to_update => $info) {
-				$status[$item_to_update] = array(
-					'name' => $info['name'],
-					'old_version' => $info['version'],
-					'new_version' => $info['update_version'],
-					'status' => $result[$item_to_update] !== null ? 'Updated' : 'Error',
-				);
+		if ( $num_to_update > 0 ) {
+			if ( ! empty( $assoc_args['format'] ) && 'summary' === $assoc_args['format'] ) {
```
Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: Would `get_flag_value()` be suitable here?

File: php/WP_CLI/CommandWithUpgrade.php [hunk 2]
```diff
@@ -264,18 +264,24 @@ protected function update_many( $args, $assoc_args ) {
...
+				foreach( $items_to_update as $item_to_update => $info ) {
+					$message = $result[$item_to_update] !== null ? 'updated successfully' : 'did not update';
```
Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: PHP Notice:  Undefined index: /var/www/subdirwp/server/static/themes/decode in php/WP_CLI/CommandWithUpgrade.php on line 270

Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: `$items_to_update` has the value of
```php
array(1) {
  ["/var/www/subdirwp/server/static/themes/p2"]=>
  array(9) {
    ["name"]=>
    string(2) "p2"
    ["status"]=>
    string(8) "inactive"
    ["update"]=>
    bool(true)
    ["update_version"]=>
    string(5) "1.5.5"
    ["update_package"]=>
    string(50) "https://downloads.wordpress.org/theme/p2.1.5.5.zip"
    ["version"]=>
    string(5) "1.4.2"
    ["update_id"]=>
    string(2) "p2"
    ["title"]=>
    string(2) "P2"
    ["description"]=>
```
MAybe `$message = $result[ $info['name'] ]`?

Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: Yes. That says: `P2 updated successfully from version 1.4.2 to version 1.5.5`

Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: @danielbachhuber Here is the fix.

Comment (danielbachhuber) [inline php/WP_CLI/CommandWithUpgrade.php]: `$info['update_id']` is what we want!

File: php/WP_CLI/CommandWithUpgrade.php (hunk 1 above)
Comment (szepeviktor) [inline php/WP_CLI/CommandWithUpgrade.php]: Would get_flag_value() be suitable here?

**Pull request merged**